### ISSUE TITLE
Wording for "Wait for navigation to complete" Step 1

### ISSUE THREAD

**Issue by vkatsikaros** [w3c/webdriver#794]
In definition [wait for navigation to complete](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-wait-for-navigation-to-complete#h-navigation):
> 1 If the current session's page loading strategy is not "none" and if there is an ongoing attempt to navigate the current browsing context that has not yet matured, wait for navigation to complete. 

This doesn't affect the output of the algorithm, but could the the last part of step 1, ie the `, wait for navigation to complete.` be re-worded to clarify that it has to do with the navigation maturing?
- In more algorithmic terms: `jump back to step 1 in this overall algorithm` or in less algorithmic, more descriptive way: a link to "wait for navigation to complete"
- In HTML terms: `wait for navigation to mature`

**Comment by shs96c** [comment]
The HTML terms seems fine to me.

**Comment by andreastt** [comment]
I agree. The HTML terms are already quite hand wavy. The actual implementation for this looks very different to the spec prose, as you can see from the draft in https://reviewboard.mozilla.org/r/113166.

**Comment by shs96c** [comment]
Fixed in 87fc5017
